pkg/apis/apis/v1alpha1/types_apiexport.go

```diff
@@ -194,6 +194,26 @@ type PermissionClaim struct {
 	// Note that one must look this up for a particular KCP instance.
 	// +optional
 	IdentityHash string `json:"identityHash,omitempty"`
+
+	// The claimed and restricted verbs for this permission claim.
+	// +required
+	Verbs []PermissionClaimVerbs `json:"verbs,omitempty"`
+}
+
+type PermissionClaimVerbs struct {
+	// The list of claimed verbs for the requested object the service provider requests.
+	// Note: this can be limited by a maximal permission policy of an underlying APIExport.
+	//
+	// +required
+	// +listType=set
+	Claimed []string `json:"claimed,omitempty"`
+
+	// The list of verbs for the requested object consumers of this permission claim are granted.
```

Comment:
I think this description or the field name could be made slightly clearer. When I see the field name `restrict`, I'm thinking that it is a list of verbs that are denied, when in actually it's defining only the allowed verbs in this case.

I'm not 100% sure of an alternate name. Perhaps `RestrictTo` (though the json `restrict_to` is ugly) or `Allow`?

Reply:
He, coincidentally me and Stefan had the same confusion :D My intuition tells me to have 1. a symmetry with PolicyRules in kube (which is what you define RBAC with and 2. explicit allow-lists only.

So, yes `RestrictTo`/`restrict_to` makes sense, thank you! I will make it clear in the godoc as well.